src/Components/Dropdowns/Composed/MenuDropdown.tsx

```diff
@@ -111,6 +111,10 @@ export const MenuDropdown: FC<MenuDropdownProps> = ({
     selectedOption
   )
 
+  useEffect(() => {
+    setSelectedItem(selectedOption)
+  }, [selectedOption])
+
```

Comment:
https://medium.com/@digruby/do-not-use-props-as-default-value-of-react-usestate-directly-818ee192f454

Reply:
does this even need to store `selectedOption` in state? Why can't it just use the value of `selectedOption` directly?

Reply:
That is a good point, let me check!